components/match2/wikis/pubgmobile/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,69 @@
+---
+-- @Liquipedia
+-- wiki=pubgmobile
+-- page=Module:GetMatchGroupCopyPaste/wiki
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local OpponentLibraries = require('Module:OpponentLibraries')
+local Opponent = OpponentLibraries.Opponent
+
+local BaseCopyPaste = Lua.import('Module:GetMatchGroupCopyPaste/wiki/Base')
+
+---WikiSpecific Code for MatchList and Bracket Code Generators
+---@class PubgmMatchCopyPaste: Match2CopyPasteBase
+local WikiCopyPaste = Class.new(BaseCopyPaste)
+
+local INDENT = WikiCopyPaste.Indent
+
+---returns the Code for a Match, depending on the input
+---@param bestof integer
+---@param mode string
+---@param index integer
+---@param opponents integer
+---@param args table
+---@return string
+function WikiCopyPaste.getMatchCode(bestof, mode, index, opponents, args)
+	local lines = Array.extend(
+		'{{Match|finished=',
+		INDENT ..
+		'|p_kill=1 |p1=10 |p2=6 |p3=5 |p4=4 |p5=3 |p6=2 |p7=1 |p8=1',
```

Comment:
why the extra line break?

also the already set values intended? (i have no idea of this game)

Reply:
> why the extra line break?

Probably line was too long on Apex, I'll change

> also the already set values intended? (i have no idea of this game)

Yeah, these are the standard values for current tournaments. It has been different in the past and may be different in the future, and there can be the odd tournament that has another set of values. But the majority of current one uses these values.